### ISSUE TITLE
$box operator causes error, while query works fine on mongo shell

### ISSUE THREAD

**Issue by rajesh-krishnan** [pyeve/eve#1122]
Thank you for developing and maintaining Eve, a wonderful package.

A query such as:  ?where={"user_location":{"$geoWithin":{"$box":[[-125.0,25.1],[-60.5,49.1]]}}}
over the REST API causes an error: Query contains unknown or unsupported operators: $box

The same query works fine in mongo shell.  Is there a reason this operator is not supported?
Adding $box to the operators set in  eve/io/mongo/mongo.py seems to fix the problem.

**Comment by nicolaiarocci** [comment]
Yes, looks like the box operator could be added without harm. Feel free to submit a PR for it, I'll gladly review it.
